standalone/PersistenceDiagramClustering/cmd/CMakeLists.txt

```diff
@@ -1,23 +0,0 @@
-cmake_minimum_required(VERSION 3.2)
```

Comment:
looks like you removed all standalones but didn't provide any replacement.
was that intended or is it just a missing ```git add```?

Reply:
The standalone has been broken ever since we changed the input from a 'multi selection thingy' to a vtkBlockDataset.
I figured I'd remove it for now, what do you think ?

Reply:
sounds for now I guess.